spk/salt-minion/Makefile

```diff
@@ -1,41 +1,73 @@
 SPK_NAME = salt-minion
-SPK_VERS = 3001.1
+ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
+SPK_VERS = 3003.2
+else
+SPK_VERS = 3000.9
+endif
 SPK_REV = 1
 SPK_ICON = src/salt-minion.png
 
-PIP = $(WORK_DIR)/../../../native/python3/work-native/install/usr/local/bin/pip
-BUILD_DEPENDS = cross/python3
-SPK_DEPENDS = "python3>=3.7"
-UNSUPPORTED_ARCHS = $(PPC_ARCHS) $(ARMv5_ARCHS)
+DEPENDS = cross/patch
+
+# DSM 6 has python2, DSM 7 has python3
+ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
+BUILD_DEPENDS  = cross/python38 cross/setuptools cross/pip
+else
+BUILD_DEPENDS  = cross/python2 cross/setuptools_py2 cross/pip_py2
+SPK_DEPENDS = "python>=2.7.11"
+endif
+BUILD_DEPENDS += cross/wheel
+BUILD_DEPENDS += cross/salt
```

Comment:
```suggestion
BUILD_DEPENDS  = cross/python38 cross/setuptools cross/pip cross/wheel  cross/salt
# DSM6 dosn't have python3 natively, we want to require our synocommunity python package
ifeq ($(call version_lt, ${TCVERSION}, 7.0),1)
SPK_DEPENDS = "python3>=3.6"
endif
```

Reply:
Since DSM6 doesn't have python3 natively, we can just require our synocommunity python package. This way you can ensure that python 3 is on the system

Reply:
Sorry it looks like you spend quite some time to work on python3 support